coderd/database/queries/notifications.sql

```diff
@@ -131,4 +135,37 @@ WHERE id IN
        WHERE nested.updated_at < NOW() - INTERVAL '7 days');
 
 -- name: GetNotificationMessagesByStatus :many
-SELECT * FROM notification_messages WHERE status = @status LIMIT sqlc.arg('limit')::int;
+SELECT *
+FROM notification_messages
+WHERE status = @status
+LIMIT sqlc.arg('limit')::int;
```

Comment:
The reason `@limit` does not work is because limit is a reserved word, and SQLc just won't let you. 

You can keep this, I've used `@limit_opt` in other places because the sql syntax checker I use can interpret that as an arg a bit easier.

Reply:
Was wondering about that, thanks!